packages/backend/src/api-impl.ts

```diff
@@ -103,6 +103,16 @@ export class BootcApiImpl implements BootcApi {
     return images;
   }
 
+  async inspectImage(image: ImageInfo): Promise<podmanDesktopApi.ImageInspectInfo> {
+    try {
+      return await podmanDesktopApi.containerEngine.getImageInspect(image.engineId, image.Id);
+    } catch (err) {
+      await podmanDesktopApi.window.showErrorMessage(`Error inspecting image: ${err}`);
+      console.error('Error inspecting image: ', err);
+    }
+    return {} as podmanDesktopApi.ImageInspectInfo;
```

Comment:
I'm not sure to understand the design like return await and return {} at the end ?

Reply:
Ditto, assuming this is for the error case and should be like a rejected promise returned to the frontend.

Reply:
updated!

Reply:
Usually pattern is throw Error or Promise<Object | undefined> not an empty object cast as the expected result